docs/snippets/common/my-addon-preset-implementation.js.mdx

```diff
@@ -2,7 +2,7 @@
 // /my-addon/src/preset.js
 
 function managerEntries(entry = []) {
-  return [...entry, require.resolve("./register")]; //👈 Addon implementation
+  return [...entry, require.resolve("./manager")]; //👈 Addon implementation
```

Comment:
@ndelangen this will double add

Reply:
We should change this documentation then.. but what will be left to document?

Perhaps this highlights that we should add a deduplication mechanism? So managerEntries never includes the same file twice?